src/groupByKey.ts

```diff
@@ -1,7 +1,19 @@
+interface Obj {}
+
 interface Item {
-  [key: string]: any;
+  [key: string]: Obj[];
 }
 
-export function groupByKey(items, field) {
-  // write code here;
+export function groupByKey(items: Obj, field: string): Item {
```

Comment:
```suggestion
export function groupByKey(items: Obj, field: keyof Item): ItemGroup {
```
Використовуй keyof (в умові задачі є підказка `string field that is one of possible item keys (field: keyof Item).`)